### PR TITLE
Remove custom support of WeChat mp articles

### PR DESCRIPTION
**Pull request by danyang685**
Current parser doesn't support the latest WeChat mp article pages, but
it supports The Open Graph protocol now

### CODE REVIEW

File: lib/onebox/engine/whitelisted_generic_onebox.rb [hunk 1]
```diff
@@ -86,6 +86,7 @@ def self.default_whitelist
           meetup.com
           mixcloud.com
           mlb.com
+          mp.weixin.qq.com
```
Comment (techAPJ) [inline lib/onebox/engine/whitelisted_generic_onebox.rb:89]: This should not be required? We allow oneboxing of all domains unless explicitly blocked.

**Comment by techAPJ** [comment]
Hi @danyang685, this PR looks great! I just added a minor comment, can you please implement the same and test again?

**Comment by danyang685** [comment]
Thank you for you reviewing. That part is removed now.

**Comment by techAPJ** [comment]
Thanks 👍

**Pull request merged**